dev-docs/bidders/9dotsmedialtd.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: bidder
+title: 9dots media LTD
+description: Prebid 9dots media LTD Bidder Adaptor
+biddercode: 9dotsmedialtd
+pbjs: false
+pbs: true
+media_types: video, banner
```

Comment:
```suggestion
media_types: video, banner, native
```

The text below claims that the adapter also supports native. In this case I would add it here as well